cli/operator/node.go

```diff
@@ -201,9 +198,9 @@ var StartNodeCmd = &cobra.Command{
 
 		validatorCtrl := validator.NewController(cfg.SSVOptions.ValidatorOptions)
 		cfg.SSVOptions.ValidatorController = validatorCtrl
-		if cfg.ReadOnlyMode {
-			cfg.SSVOptions.DutyExec = duties.NewReadOnlyExecutor(Logger)
-		}
+		//if cfg.ReadOnlyMode {
```

Comment:
why not removing?

Reply:
You can lookup `p2p host was configured` log,
I've added it as a field (`selfPeer`) to all p2p logs